infor-design/enterprise-ng
# Misc - Add new types and 4.31 features

**Explain the _details_ for making this change. What existing problem does the pull request solve?**

Added all new 4.31 types and changed the masthead code.

**Related github/jira issue (required)**:
Misc

**Steps necessary to review your pull request (required)**:
- tests now pass
- inspect types for logical errors
- smoke test
- notice the new masthead updates with a picker and try it in both subtle and vibrant mode
